vuu-ui/showcase/src/examples/Apps/NewTheme.examples.tsx

```diff
@@ -32,6 +32,23 @@ let displaySequence = 1;
 
 export const ShellWithNewTheme = () => {
   const [dialogContent, setDialogContent] = useState<ReactElement>();
+
+  const handleCloseDialog = useCallback(() => {
+    setDialogContent(undefined);
+  }, []);
+
+  const handleSave = useCallback((layoutName: string) => {
+    console.log("Save layout as " + layoutName);
+  }, []);
+
+  const handleScreenshot = async () => {
+    await takeScreenshot(
+      document.getElementsByClassName("vuuShell-content")[0] as HTMLElement
```

Comment:
I'm going to change the selector for the node passed in to something more reliable.

Reply:
[aa898c2](https://github.com/ScottLogic/finos-vuu/pull/23/commits/aa898c20d1ff7f17315d89bf0996fa47f13c5518)